sdk/monitor/monitor-query/test/public/logsQueryClient.spec.ts

```diff
@@ -352,20 +310,19 @@ describe("LogsQueryClient live tests", function() {
         testRunId = env.TEST_RUN_ID;
       } else {
         testRunId = `ingestedDataTest-${Date.now()}`;
-
         // send some events
-        await runWithTelemetry(this, (provider) => {
-          const tracer = provider.getTracer("logsClientTests");
-
-          tracer
-            .startSpan("testSpan", {
-              attributes: {
-                testRunId,
-                kind: "now"
-              }
-            })
-            .end();
-        });
+        //  await runWithTelemetry(this, (provider) => {
```

Comment:
why commented?

Reply:
I have disabled the tests using opentelemetry-exporter for data ingestion - bcz there is a bug in that package - also bcz it doesn't support browser. We don't really need tests to do the data ingestion. So commenting it out for now.